docs/wmco-prerequisites.md

```diff
@@ -0,0 +1,45 @@
+# Cluster and OS Pre-Requisites
+Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
+may be relevant.
+
+## Supported Kubernetes Distribution Version
```

Comment:
`Supported OpenShift versions`, though I am thinking this section can be taken out given the table below

Reply:
Should we keep it so people know which version is best for their use case? (configuring a VM vs. BYOH Instance)

Reply:
Once we complete BYOH support we can add a separate section for BYOH.